src/web/mjs/connectors/MangaDex.mjs

```diff
@@ -11,7 +11,17 @@ export default class MangaDex extends Connector {
         this.url = 'https://mangadex.org';
         this.api = 'https://api.mangadex.org';
         this.requestOptions.headers.set('x-referer', this.url);
-        this.throttleGlobal = 100;
+        this.requestOptions.headers.set('x-sec-ch-ua', '" Not A;Brand";v="99", "Chromium";v="96", "Google Chrome";v="96"');
+        this.config = {
+            throttle: {
```

Comment:
`throttle` is a magic key which will be used when as delay when downloading images. Should be renamed to something else.

Reply:
i mean this should apply to downloads no?

Reply:
nvm downloading dosnt actually do any requests to the api